web-api/tests/personalized_documents.rs

```diff
@@ -260,10 +319,41 @@ fn personalization_include_properties(include_properties: bool) {
 
 #[test]
 fn test_personalization_include_properties() {
-    personalization_include_properties(true);
+    personalization_include_properties(true, false);
 }
 
 #[test]
 fn test_personalization_exclude_properties() {
-    personalization_include_properties(false);
+    personalization_include_properties(false, false);
+}
+
+#[test]
+fn test_deprecated_personalization_include_properties() {
+    personalization_include_properties(true, true);
+}
+
+#[test]
+fn test_deprecated_personalization_exclude_properties() {
+    personalization_include_properties(false, true);
+}
```

Comment:
Integration tests are slow to start up, so where viable we likely want to have one test which covers more part instead of many tests which cover one part.

Through only if viable and no inference is expected/likely/possible.

But what we do here is somewhat one the line of unit testing with integration tests.

Sadly this is one of the many things where rust build-in testing lack support (I mean it's also on of the things where theoretical preference of maximal isolation diverges from the reality  of having limited resources and time.

Reply:
i'll set the tests up in a more compact way, this should solve it.